src/components/Allbooks.js

```diff
@@ -1,10 +1,10 @@
 /* eslint-disable react/prop-types */
```

Comment:
Your project looks great. As much as possible, kindly avoid disabling linter checks in line. This is because they represent coding standards that you share with your team and most of the time, enforce best coding practices.